File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int res = 0;
        for (string str: details) {
            if (str[11] > '6' || (str[11] == '6' && str[12] >= '1')) {
                res++;
            }
        }
        return res;
    }
};

File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int seniors = 0;
        for (String detail: details) {
            char ch1 = detail.charAt(11), ch2 = detail.charAt(12);
            if (ch1 > '6' || (ch1 == '6' && ch2 > '0')) {
                seniors += 1;
            }
        }
        return seniors;
    }
}